Mathlib/CategoryTheory/Limits/Cones.lean

```diff
@@ -676,155 +701,156 @@ variable {F : J ⥤ C} {G : J ⥤ C} (H : C ⥤ D)
 
 open CategoryTheory.Limits
 
+/- Porting note: dot notation on the functor is broken for `mapCone` -/
```

Comment:
Please move this note next to a manifestation of the problem.

Reply:
👍